tests/createProduct.test.js

```diff
@@ -0,0 +1,150 @@
+const request = require('supertest')
+const { User, Product } = require("../models/index")
+const app = require("../app")
+const { sequelize } = require("../models/index")
+const { queryInterface } = sequelize
+let { createToken } = require("../helpers/accesToken")
+let acces_token;
+let acces_tokenCustomer;
+
+afterAll(done => {
+    queryInterface.bulkDelete('Products', null, {})
+        .then(data => {
+        return queryInterface.bulkDelete("Users", null, {})         
+        })
+        .then(data => {
+            done()
+        })
+        .catch(err => {
+            done(err)
+        })
+    
+})
+    
+beforeAll(done => {
+    let objAdmin = {
+        email: "admin@mail.com",
+        password: "qwerty",
+        role: "admin"
+    }
+
+    let objCustomer = {
+        email: "febri@mail.com",
+        password: "qwerty",
+        role: "customer"
+    }
+    
+    User.create(objAdmin)
+        .then(data => {
+            acces_token = createToken({ data })
+            return User.create(objCustomer)
+        })
+        .then(data => {
+            acces_tokenCustomer = createToken({data})
+            done()
+        })
+        .catch(err => {
+            done(err)
+        })
+})
+
+
+describe("POST /products", () => {
+
+    // berhasil add create
+    test("Succes to add product /product", (done => {
+        request(app)
+        .post("/products/")
+        .set("acces_token", acces_token)
+        .send({
+            name: "masker",
+            image_url: "cek",
+            price: 1200000,
+            stock: 6
+        })
+        .end(function (err, res) {
+            if(err) {
+                done(err)
+            }else {
+                const { body, status } = res
+                expect(body).toHaveProperty("name", "masker")
+                expect(body).toHaveProperty("price", 1200000)
+                expect(body).toHaveProperty("image_url", "cek")
+                expect(body).toHaveProperty("stock", 6)
+                expect(status).toBe(201)
+                done()
+            }
+        })
+    }))
+
+    // tidak punya token
+    test("Failed to add product /product", (done => {
```

Comment:
tidak punya token nya dikasih tau di deskripsi test nya (param 1 yg string) misal Failed to add product /product because not give access_token as header